site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -127,7 +153,7 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
                 ],
             });
         } else {
-            if (stopToAdd) {
+            if (stopToAdd && (pageState.inputs.stops ? pageState.inputs.stops.length < 100 : true)) {
```

Comment:
Should this be <= 100 as 100 is the limit?

Reply:
don't want to add anything if we are already at 100 - if it is 99 we want to add a stop

Reply:
ah okay - just realised we have a boolean mixed with an inline if. It could be simplified to `stopToAdd && pageState.inputs.stops && pageState.inputs.stops.length < 100`

Reply:
done